dcos_installer/config.py

```diff
@@ -45,6 +45,13 @@ def normalize_config_validation(messages):
     return validation
 
 
+def normalize_config_validation_exception(exception):
```

Comment:
Please add a docstring for this function.

❓ I think that it is worth having a unit test for this function.